src/index.js

```diff
@@ -3,6 +3,16 @@ import chProcess from 'child_process';
 import _ from 'lodash';
 import fs from 'fs';
 import chokidar from 'chokidar';
+import path from 'path';
+
+let pathSet = false;
+function setPath() {
+  if(!pathSet) {
```

Comment:
Okay, so inlining doesn't work. Also, good catch on `pathSet` not being used, I fixed that. I think I'm going to try to make `setPath` and IIFE and see how that works out.